.env.example

```diff
@@ -12,9 +12,9 @@ PK_FUEL_SIGNER1=
 PK_FUEL_SIGNER2=
 
 # Settlement layer contract deployments
-FUEL_SIDECHAIN_CONSENSUS_ADDRESS=
+FUEL_CHAIN_CONSENSUS_ADDRESS=
 FUEL_MESSAGE_PORTAL_ADDRESS=
-L1_ERC20_GATEWAY_ADDRESS=
+FUEL_ERC20_GATEWAY_ADDRESS=
```

Comment:
We use the term "ERC20" on the ethereum contracts, because it's the ethereum contracts and we're clearly interacting with the ERC20 standard. The "ERC20" in the name just describes what the gateways niche is. There can be other gateways for other things later